mavros_msgs/msg/Tunnel.msg

```diff
@@ -0,0 +1,22 @@
+# Tunnel
+#
+# https://mavlink.io/en/messages/common.html#TUNNEL
+
+uint8 target_system
+uint8 target_component
+uint16 payload_type
+uint8 payload_length
+uint8[128] payload
```

Comment:
For ROS2 it's possible to set upper limit. For ROS1 i suppose better to keep sending that unneeded payload tail.

Reply:
Sounds fair. (I'm looking forward to migrating my stuff to ROS2 at some point...)